mopidy_local/translator.py

```diff
@@ -6,17 +6,19 @@
 logger = logging.getLogger(__name__)
 
 
-def local_uri_to_file_uri(uri, media_dir):
+def local_uri_to_file_uri(local_uri, media_dir):
     """Convert local track or directory URI to file URI."""
-    return path_to_file_uri(local_uri_to_path(uri, media_dir))
+    return path_to_file_uri(local_uri_to_path(local_uri, media_dir))
```

Comment:
You could also replace part of this with a simple `.as_uri()`.

Reply:
Done